employee_reviews.rb

```diff
@@ -0,0 +1,32 @@
+require './department'
+require './employee'
+#
+# oliver = Employee.new( "Oliver Twist", 1.0 )
```

Comment:
In general, it is best to remove all commented code before committing.  If you need to recover it, git's history will make that possible.